### PR TITLE
Enhance Certificate Display and Download Functionality

### PR DESCRIPTION
**Pull request by TanmayDhobale**
### PR Fixes:  
- 1  This pull request introduces several enhancements to the certificate display component . It adjusts the visual presentation of certificates and updates the functionality to include additional information such as the certificate ID and verification details. The changes aim to improve user experience by making certificates more informative and visually appealing.

## Changes Made
- **Visual Enhancements**: Updated the certificate display to center the certificate image and information on the page. Increased the size of the image for better visibility.
- **Text Overlay**: Added dynamic text overlays to the certificate image, including the user's name, course title, certificate ID, and a verification email. These overlays are styled to be bold and black for better contrast and readability.
- **Download Functionality**: Updated the button for downloading the certificate. and user can shere on linkdin and on X
-** Not everyone can download the certificate. Only those who have completed the course and fulfilled its requirements will be able to generate the certificate.

Resolves #[ #483 ] 

### Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by TanmayDhobale** [comment]
Blue Minimalist Certificate Of Achievement (1)
@hkirat  better ?

**Comment by nimit9** [comment]
@hkirat @TanmayDhobale

**Comment by TanmayDhobale** [comment]
@hkirat yooo. can u review this one ???

**Comment by hkirat** [comment]
reviewing

File: src/utiles/generateCertificate.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    });
+
+    const pdfBytes = await pdfDoc.save();
+    const certificatePath = path.join(
+      process.cwd(),
+      'public',
+      'certificates',
+      `${data.certificateId}.pdf`,
+    );
+    await fs.promises.writeFile(certificatePath, pdfBytes);
```
Comment (hkirat) [inline src/utiles/generateCertificate.ts:73]: Storing it on the server?
Bad idea

Should ideally just be generated on the fly and returned to the user
Or should go to S3

Comment (TanmayDhobale) [reply src/utiles/generateCertificate.ts:73]: got it my bad , so should i make change or close this pr ?

Comment (hkirat) [reply src/utiles/generateCertificate.ts:73]: can u put this design on what is merged in master?

Comment (TanmayDhobale) [reply src/utiles/generateCertificate.ts:73]: sure

**Pull request closed without merging**